Palindrome.UnitTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Palindrome.UnitTests
+{
+    [TestClass]
+    public sealed class PalindromeUnitTests
+    {
+        /* 
+        *  Test class contains 3 helper methods.
+        *  Passing 6/6 test cases.
+        */
+
+        /// <summary>
+        /// Generates a random string of upper and lower case letters including ' '.
+        /// Passes result string to RandomPalindromeGenerator.
+        /// </summary>
+        /// <returns>A string containing less than 1000 characters.</returns>
+        private static string RandomString()
+        {
+            Random rand = new Random();
```

Comment:
Due to how random numbers are seeded, you should create a single instance of the `Random` class at the top of the class. I can explain more at another time.

```csharp
private readonly Random Random = new Random();
```